server/tracedb/awsxray.go

```diff
@@ -0,0 +1,801 @@
+package tracedb
+
+import (
```

Comment:
Big chunky file that contains everything related to the x-ray connectivity and parsing of the data

Reply:
Makes sense to create a package for XRay inside of `tracedb` and break this file by responsibilities/concerns?